.dagger/src/index.ts

```
@@ -0,0 +1,1258 @@
+      cleanData = jsonMatch[0]
+
+      try {
+        const result = JSON.parse(cleanData)
+
+        // Try extracting from `.returns.deployed.value` (primary method)
+        if (
+          result.returns &&
+          result.returns.deployed &&
+          result.returns.deployed.value
+        ) {
+          contractAddress = result.returns.deployed.value
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)